ba/js/lesson3/script.js

```diff
@@ -0,0 +1,196 @@
+//Task 1
+
+let start = prompt(" Enter start number");
+let end = prompt("Enter end number");
+let sum = 0;
+for (let i = +start; i < +end + 1; i++) {
+  sum += i;
+}
+
+alert(`Total: ${sum}`);
+//Task 2
+let numA = prompt(" Enter number A");
+let numB = prompt("Enter number B");
+
+let a = +numA;
+let b = +numB;
+
+while (a != 0 && b != 0) {
+  a > b ? (a = a % b) : (b = b % a);
+}
+
+alert(`GCD of ${numA} and ${numB} is ${a + b} `);
+
+//Task 3
+dividers = [];
+let number = prompt("Enter a number");
+
+for (let i = Math.trunc(number / 2); i > 0; i--) {
+  if (number % i == 0) {
+    dividers.push(i);
+  }
+}
+
+alert(dividers);
+
+//Task 4
+
+let number_ = prompt("Enter a number...");
+console.log(number_.length);
+
+//Task 5
+
+let positive = 0;
+let negative = 0;
+let zero = 0;
+let odd = 0;
+let even = 0;
+console.log(positive, negative);
+for (let i = 0; i < 10; i++) {
+  let num = prompt("Enter a number....");
+  num = +num;
+
+  num == 0 ? zero++ : num > 0 ? positive++ : negative++;
+  num % 2 ? odd++ : even++;
+}
+
+console.log(` You have entered:
+Positive numbers: ${positive}
+Negative numbers: ${negative}
+Zeroes:           ${zero}
+Odd numbers:      ${odd}
+Even numbers:     ${even}`);
+
+//Task 6
+do {
+  let firstNum = +prompt("Enter a number: ");
+  let secondNum = +prompt("Enter another number: ");
+  let operationSign = prompt("Enter the operation sign (+ - * /): ");
+  switch (operationSign) {
+    case "+":
+      console.log(
+        `${firstNum} ${operationSign} ${secondNum} = ${firstNum + secondNum}`
+      );
+      break;
+    case "-":
+      console.log(
+        `${firstNum} ${operationSign} ${secondNum} = ${firstNum - secondNum}`
+      );
+      break;
+    case "*":
+      console.log(
+        `${firstNum} ${operationSign} ${secondNum} = ${firstNum * secondNum}`
+      );
+      break;
+    case "/":
+      console.log(
+        `${firstNum} ${operationSign} ${secondNum} = ${firstNum / secondNum}`
+      );
+      break;
+    default:
+      break;
+  }
+  var continueCalc = prompt(
```

Comment:
do not use var

Reply:
I had to use a global variable otherwise, the variable  "continueCalc" is not initialized in the loop condition.